docs/API/content-type/listing-co.md

```diff
@@ -1,61 +1,991 @@
+| notEquals          | Object parameter must not be equal to `filter`, can be used with string type parameters                                                                       |
+| contains           | Object parameter must contain `filter`, can be used with every type parameters                                                                                |
+| notContains        | Object parameter must not contain `filter`, can be used with every type parameters                                                                            |
+| startsWith         | Object parameter must start with `filter`, can be used with string type parameters                                                                            |
+| endsWith           | Object parameter must end with `filter`, can be used with string type parameters                                                                              |
+| lessThanOrEqual    | Object parameter must be less or equal to `filter`, can be used with number type parameters                                                                   |
+| lessThan           | Object parameter must be less than a filter, can be used with number type parameters                                                                          |
+| greaterThanOrEqual | Object parameter must be greater or equal than `filter`, can be used with number type parameters                                                              |
+| greaterThan        | Object parameter must be greater than `filter`, can be used with number type parameters                                                                       |
+| inRange            | Object parameter must be between `filter` and `filter2`, it is only filter type that has three keys in filter object, can be used with number type parameters |
+
+!!! Example
+
+    ```
     {
-      "id": "123123123",
-      "title": "New object",
-      "postContent": "This will be the new <b>content</b>",
-      "internal": {
-        "createdAt": "2019-10-25T20:22:37+00:00",
-        "deletedAt": "",
-        "updatedAt": "2019-10-30T20:32:42+00:00",
-        "contentType": "blogposts"
-      }
+        filters: {
+            title: {
+                type: "equals",
+                filter: "New object"
+            },
+            postContent: {
+                type: "contains",
+                filter: "content"
+            }
+        }
     }
-  ]
-}
-```
+    ```
+
+    === "CURL"
+    
+        ```
+        curl -X GET "https://api.flotiq.com/api/v1/content/blogposts?page=1&limit=20&order_by=internal.createdAt&order_direction=asc&hydrate=0&filters=%7Btitle%3A%7Btype%3A%22equals%22%2Cfilter%3A%22New%20object%22%7D%2CpostContent%3A%7Btype%3A%22contains%22%2Cfilter%3A%22content%22%7D%7D" -H "X-AUTH-TOKEN: YOUR_API_KEY" -H "accept: application/json"
```

Comment:
When I copy and run this example I get the error "malformed filters"

Reply:
This string decoded contains `"{title:{type:"equals",filter:"New object"},postContent:{type:"contains",filter:"content"}}"`
It should be JSON, with object keys quoted